### PR TITLE
PLAT-140537: Add spotlightId in InputField to support VKB reopen

### PR DESCRIPTION
**Pull request by changgilee**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
InputField has to re-focused by app when "show password" checkbox is clicked in Input component

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Add spotlightId to InputField

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
N/A

### Links
[//]: # (Related issues, references)
PLAT-140537

### Comments
N/A

### CODE REVIEW

**Review by 0x64 — APPROVED** [review]
:lgtm:

File: Input/Input.js [hunk 1]
```diff
@@ -73,6 +73,14 @@ const InputPopupBase = kind({
 		 */
 		disabled: PropTypes.bool,
 
+		/**
+		 * Set spotlightId in InputField
```
Comment (MikyungKim) [inline Input/Input.js:77]:
```suggestion
		 * Set spotlightId to InputField.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 - `sandstone/Picker` and `sandstone/RangePicker` public class names `title` and `inlineTitle`
 - `sandstone/Scroller` and `sandstone/VirtualList` prop `hoverToScroll` to scroll by hover
 - `sandstone/VirtualList` prop `snapToCenter`
+- `sandstone/Input` prop `inputFieldSpotlightId` to set `spotlightId` in `InputField`
```
Comment (MikyungKim) [inline CHANGELOG.md:13]:
```suggestion
- `sandstone/Input` prop `inputFieldSpotlightId` to set `spotlightId` to `InputField`
```

Comment (MikyungKim) [reply CHANGELOG.md:13]: Please put this line first in the Added section since it starts with I.

File: Input/Input.js [hunk 3]
```diff
@@ -73,6 +73,14 @@ const InputPopupBase = kind({
 		 */
 		disabled: PropTypes.bool,
 
+		/**
+		 * Set spotlightId to InputField.
```
Comment (MikyungKim) [inline Input/Input.js:77]:
```suggestion
		 * Sets spotlightId to InputField.
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**